<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_Multi">
       <property name="title">
        <string>Value</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radio_Fixed">
          <property name="text">
           <string>Single</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_Normal">
          <property name="text">
           <string>Normal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_Linearity">
          <property name="text">
           <string>Linearity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_Gauss">
          <property name="text">
           <string>Gauss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_Possion">
          <property name="text">
           <string>Possion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_Exponential">
          <property name="text">
           <string>Exponential</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="group_randomPara">
     <property name="title">
      <string>Random Para</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1,2">
      <item>
       <widget class="QLabel" name="label_Para1">
        <property name="text">
         <string>Para1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_Para1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_Para2">
        <property name="text">
         <string>Para2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_Para2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ParaSet">
     <property name="title">
      <string>Para Set</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_Value">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_Value"/>
      </item>
      <item>
       <widget class="QLabel" name="label_ParaFrom">
        <property name="text">
         <string>Para From:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_paraFrom"/>
      </item>
      <item>
       <widget class="QLabel" name="label_ParaTo">
        <property name="text">
         <string>Para To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_paraTo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_ParaNumber">
        <property name="text">
         <string>Para Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_paraNum">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Random">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="out_OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="out_CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>out_OkButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>out_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
